soundfile.py

```diff
@@ -12,17 +12,39 @@
 
 import os as _os
 import sys as _sys
+import numpy as np
+import numpy.typing as npt
```

Comment:
no abbreviations, please, if they can be avoided.

Reply:
Fixed in https://github.com/bastibe/python-soundfile/pull/462/commits/a8a77c1e8155dc029f53aad1d69a7a501a60a671